apps/apps.humanatlas.io/src/app/pages/us2/tissue-origin-predictor/tissue-origin-predictor.component.scss

```diff
@@ -19,9 +21,9 @@
 
 .info-text {
   margin-top: 3.5rem;
-  font: var(--mat-sys-body-large);
-  letter-spacing: var(--mat-sys-body-large-tracking);
-  color: var(--mat-sys-on-secondary-fixed);
+  font: vars.$body-large;
+  letter-spacing: vars.$body-large-tracking;
+  color: vars.$on-secondary-container;
```

Comment:
Please make sure the switch from `on-secondary-fixed` to `on-secondary-container` is correct